packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -313,10 +315,24 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [theme, plugins]);
 
+  useEffect(() => {
+    const themeDirectionValue = themeDirection(theme);
+    // check that editor exists and theme was actually changed
+    if (editor.current && previousThemeDirection.current !== themeDirectionValue) {
+      const format = getFormatState(editor.current);
+      if (format.direction !== themeDirectionValue) {
```

Comment:
Isn't checking format.direction already sufficient? Do we need to introduce  previousThemeDirection?

Reply:
Unfortunately, it's not sufficient as format.direction is set only after set direction function is called and that's why we need to updarstand if this a real change of theme direction or it's just an initial theme value for this component

Reply:
One of the issues about calling `setDirection` when setting the initial value of theme is that `setDirection` uses focus call inside and it won't work very well with our `autoFocus` prop

Reply:
> Unfortunately, it's not sufficient as format.direction is set only after set direction function is called and that's why we need to upderstand if this a real change of theme direction or it's just an initial theme value for this component

Should we add a comment here to document this information? I feel it's quite useful to remember this finding.